### PR TITLE
Resolves #2523 - New spenddown report

### LINKED ISSUE #2523: New Script - Report about MAXIS Cases with a spenddown
Need a script to identify cases that appear to have a spenddown in MAXIS. Will also gather information about overages in the budget, elig type and standard and indicator in MMIS. 

I have this.
Will pull. 
Thank you

### PR DESCRIPTION
**Pull request by C-Love**
Blip - 'New script that will create an Excel spreadsheet listing information about cases that appear to have a spenddown in MAXIS.'

### CODE REVIEW

File: Script Files/BULK/BULK - SPENDDOWN REPORT.vbs [hunk 1]
```diff
@@ -0,0 +1,637 @@
...
+		transmit
+	Else
+		attn
+		EMConnect "B"
+		attn
+		EMReadScreen MMIS_B_check, 7, 15, 15
+		If MMIS_B_check <> "RUNNING" then 
+			MMIS_checkbox = unchecked  
+			script_continue = MsgBox ("MMIS does not appear to be running." & vbNewLine & "Do you wish to have the report without the MMIS Spenddown Indicator checked?", vbYesNo + vbQuestion, "MMIS not running")
+			IF script_continue = vbNo Then script_end_procedure
```
Comment (CDPotter) [inline Script Files/BULK/BULK - SPENDDOWN REPORT.vbs]: Script end procedure is missing arguments.

File: Script Files/BULK/BULK - SPENDDOWN REPORT.vbs [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+	EMFocus 'Bringing window focus to the second screen if needed.
+
+	'Sending MMIS back to the beginning screen and checking for a password prompt
+	Do 
+		PF6
+		EMReadScreen password_prompt, 38, 2, 23
+	  	IF password_prompt = "ACF2/CICS PASSWORD VERIFICATION PROMPT" then 
+		  	MMIS_checkbox = unchecked  
+		  	script_continue = MsgBox ("MMIS does not appear to be running." & vbNewLine & "Do you wish to have the report without the MMIS Spenddown Indicator checked?", vbYesNo + vbQuestion, "MMIS not running")
+		  	IF script_continue = vbNo Then script_end_procedure
```
Comment (CDPotter) [inline Script Files/BULK/BULK - SPENDDOWN REPORT.vbs]: Script end procedure is missing arguments.

File: Script Files/BULK/BULK - SPENDDOWN REPORT.vbs [hunk 3]
```diff
@@ -0,0 +1,637 @@
...
+
+
+'Autofitting columns
+For col_to_autofit = 1 to 12
+	ObjExcel.columns(col_to_autofit).AutoFit()
+Next
+
+'Logging usage stats
+STATS_counter = STATS_counter - 1                      'subtracts one from the stats (since 1 was the count, -1 so it's accurate)
+script_end_procedure("")
```
Comment (CDPotter) [inline Script Files/BULK/BULK - SPENDDOWN REPORT.vbs]: Can we add a I'm done message box?

**Comment by C-Love** [comment]
Certainly. I will pull that in shortly

**Comment by C-Love** [comment]
@CDPotter - here is that message. I also realized that I left my stats updates out.

**Comment by CDPotter** [comment]
Thank you! Looks like the script_end_procedures on 489 and 507 are missing their function arguments.

**Comment by C-Love** [comment]
@CDPotter  - all set - thank you for your keen eyes.

**Comment by CDPotter** [comment]
I'm getting a merge conflict with the main menu script. Could you resolve and re-pull?

**Comment by C-Love** [comment]
@CDPotter  - can you either resolve this or ignore that file and I will create a seperate pull request for this on Monday? I submitted this days ago and now I'm home sick, I don't have atom and the internet based approach does not give me the tools I need to resolve this. I have replaced the entire file with what is currently on Master so I have no idea where the conflict is coming from. We really need this released this month.

**Comment by CDPotter** [comment]
It can wait till Monday. Since this pull is already in progress we will make sure it gets into release!

**Pull request closed without merging**